src/matter/session/secure/PaseMessages.ts

```diff
@@ -6,40 +6,53 @@
 
 import { BooleanT, ByteStringT, Field, ObjectT, OptionalField, UInt16T, UInt32T } from "../../../codec/TlvObjectCodec";
 
+import {
+    CRYPTO_HASH_LEN_BYTES,
+    CRYPTO_PUBLIC_KEY_SIZE_BYTES
+} from "../../../crypto/Crypto";
+import { MatterCoreSpecificationV1_0 } from "../../../Specifications";
+
+/** @see {@link MatterCoreSpecificationV1_0} § 2.12.5 */
+const SedParametersT = ObjectT({
+    /** Maximum sleep interval of node when in idle mode. */
+    idleRetransTimeoutMs: OptionalField(1, UInt32T), /* default: 300ms */
+    /** Maximum sleep interval of node when in active mode. */
+    activeRetransTimeoutMs: OptionalField(2, UInt32T), /* default: 300ms */
+});
+
+/** @see {@link MatterCoreSpecificationV1_0} § 4.13.1.2 */
 export const PbkdfParamRequestT = ObjectT({
-    random: Field(1, ByteStringT()),
-    sessionId: Field(2, UInt16T),
-    passcodeId: Field(3, UInt16T),
+    random: Field(1, ByteStringT({ length: 32 })),
+    sessionId: Field(2, UInt16T), // Specs: range: 16bits
+    passcodeId: Field(3, UInt16T), // Specs: length: 16bits so min is 0x1000 ??
```

Comment:
Any idea on that? ;-)

Reply:
I don't think we have to worry about that. When used, 1 will be converted to 0x0001.

Reply:
Ok, lets see how it will end ... my interpretation of "length 16 bit" was that at least 16 needs to be used .. this in theory brings me to min 0x8000 (uups typo in comment) aka 32.768 as "smallest number that is actually using 16 bit in length